docs/bootstrap.md

```diff
@@ -51,7 +51,7 @@ bootstrap(); // actually run the async function
 
 ## Create HTTP GraphQL endpoint
 
-In most cases, the GraphQL app is served by a HTTP server. After building the schema we can create it using e.g. [`apollo-server`](https://github.com/apollographql/apollo-server) package:
+In most cases, the GraphQL app is served by a HTTP server. After building the schema we can create it using the [`apollo-server`](https://github.com/apollographql/apollo-server) package:
```

Comment:
`apollo-server` is just one of the tools that you can use - earlier the examples used `graphql-yoga`. I would like to underline that fact.